common/types.go

```diff
@@ -335,30 +624,30 @@ func (a Address) Format(s fmt.State, c rune) {
 
 // SetBytes sets the address to the value of b.
 // If b is larger than len(a), b will be cropped from the left.
-func (a *Address) SetBytes(b []byte) {
+func (a *InternalAddress) SetBytes(b []byte) { // May want to check scope here
```

Comment:
Is checking scope not needed ? or still needs to be decided ?

Reply:
Function is not private, so scope checking is not needed